ocs_ci/ocs/ui/views.py

```diff
@@ -2653,6 +2653,46 @@
         ".toast-notifications-list-pf .alert-success",
         ".co-alert--success",
     ],
+    "block_public_access_tab": (
+        "//button[contains(., 'Block public access')]",
+        By.XPATH,
+    ),
+    "manage_public_access_settings_button": (
+        "//button[normalize-space()='Manage public access settings']",
+        By.XPATH,
+    ),
+    "block_all_public_access_checkbox": (
+        "//label[.//p[contains(., 'Block all public access')]]//input[@type='checkbox']",
+        By.XPATH,
+    ),
+    "block_all_public_access_msg": (
+        "//label[.//p[contains(., 'Block all public access')]]//span[contains(@class,'pf-v5-c-label__text')]",
+        By.XPATH,
+    ),
+    "block_new_public_policies_checkbox": (
+        "//label[.//p[contains(., 'granted through new public bucket policies')]]//input[@type='checkbox']",
+        By.XPATH,
+    ),
+    "block_new_public_policies_msg": (
+        "//label[.//p[contains(., 'new public bucket policies')]]//span[contains(@class,'pf-v5-c-label__text')]",
+        By.XPATH,
+    ),
+    "block_cross_account_checkbox": (
+        "//label[.//p[contains(., 'cross-account access')]]//input[@type='checkbox']",
+        By.XPATH,
+    ),
+    "block_cross_account_msg": (
+        "//label[.//p[contains(., 'cross-account access')]]//span[contains(@class,'pf-v5-c-label__text')]",
```

Comment:
please do not use pf-v5 or any prefixes starring with pf. These class attributes are changing from version to version.
Since you are already using "contains" function, you can just remove the prefix "pf-v5-".

Reply:
@DanielOsypenko Done, thank you.